src/context/token-repository.ts

```diff
@@ -26,6 +27,16 @@ export class TokenRepository extends BaseRepository<TokenEntity> {
           tokenIds
         });
       }
+
+      if (name) {
+        const nameLike = name.replace(/\*/g, "%");
+        if (nameLike !== name) {
```

Comment:
Regular expressions are costly in terms of performance, and we are potentially doing that twice in the same request. That's not a big deal now, but we need to be prepared to growth. These little improvements summed can do a huge difference in the future with a huge load of requests.

That being said, I would suggest you to replace this verification by something like `name.indexOf("*") !== -1` and only try to replace `*` by `%` if `true`

Reply:
Done, exiting point of view!